Nicanor008/Store-Manager-API
# #161488528 feedback implementation

**What this PR do?**
It implements the feedback given 

**Brief Description**
The following are the completed tasks:
1. Empty fields to give JSON response on registration
2. All responses be in JSON format
3. Message response product GET request
4. Set token duration to 1 hour
5. Use base url in tests

**How to test**
clone this project and open it
activate the virtual environment
activate the server `flask run` and run the `nosetests` to check if all tests are passing
test the url in postman

**Related Pivotal Tracker Board**
#161488528
